django/core/management/__init__.py

```diff
@@ -254,14 +254,12 @@ def fetch_command(self, subcommand):
         try:
             app_name = commands[subcommand]
         except KeyError:
-            if os.environ.get("DJANGO_SETTINGS_MODULE"):
-                # If `subcommand` is missing due to misconfigured settings, the
-                # following line will retrigger an ImproperlyConfigured exception
-                # (get_commands() swallows the original one) so the user is
-                # informed about it.
-                settings.INSTALLED_APPS
+            if os.environ.get("DJANGO_SETTINGS_MODULE") and self.settings_exception:
+                sys.stderr.write(str(self.settings_exception) + "\n")
```

Comment:
```suggestion
                sys.stderr.write(self.style.NOTICE(str(self.settings_exception) + "\n"))
```
And have the init set `self.style = color_style()` and update the others using a style